Models/models.js

```diff
@@ -9,13 +9,28 @@ exports.selectCategories = () => {
     })
 }
 
-exports.selectReviews = () => {
-    return db.query(
+exports.selectReviews = (order = 'DESC', sort_by = 'created_at', category) => {
+    if (!['ASC', 'DESC'].includes(order.toUpperCase())){
+        order = 'DESC'
+    }
+    if (!['owner', 'review_id', 'category', 'review_img_url', 'created_at', 'desginer', 'comment_count']){
+        sort_by = 'created_at'
+    }
```

Comment:
what is this if statement actually checking for? do you actually want to change sort_by to create_at here? or would you want to throw an error if sort_by is not what you expect it to be?